tests/util_tests/test_process_file.py

```diff
@@ -29,29 +30,15 @@ def test_success(db):
     db.session.add(file)
     db.session.commit()
 
-    while_processing_started_at: Optional[datetime.datetime] = None
```

Comment:
All this stuff was removed because we no longer update the `file` object before calling `process`.